src/main/java/Sorting.java

```diff
@@ -0,0 +1,273 @@
+
+
+import java.util.*;
+
+/**
+ * Created by sam on 6/17/16.
+ */
+public class Sorting {
+
+    // implement bubble sort
```

Comment:
New game plan (sorry about previous comment). Make this into Javadoc that describes the algorithm (briefly). Give me a big-O for avg runtime in the comment too!